src/test/java/org/opentripplanner/netex/mapping/StopAndStationMapperTest.java

```diff
@@ -157,6 +159,7 @@ public void mapStopPlaceAndQuays() {
       MappingSupport.ID_FACTORY,
       quaysById,
       null,
+      StopModel.of(),
```

Comment:
Having the smoke test is good but we really shouldn't rely to much on it and use unit tests. In this case it should be easy to write some. Can you please add some?